src/index.js

```diff
@@ -0,0 +1,77 @@
+import './style.css';
+import ToDoList from './modules/ToDoList.js';
+import Task from './modules/Task.js';
+import createTaskDOM from './modules/taskEvents.js';
+import * as storage from './modules/localStorageFunctions.js';
+
+const toDoList = new ToDoList();
+toDoList.taskList = storage.load('tasks');
+
+const taskList = document.querySelector('.to-do-list');
+
+function addCheckBoxListener(checkbox) {
+  const checkboxes = document.querySelectorAll('.checkbox');
+  checkbox.addEventListener('change', () => {
+    const boxIndex = Array.prototype.indexOf.call(checkboxes, checkbox);
+    toDoList.taskList[boxIndex].completed = !toDoList.taskList[boxIndex].completed;
+    storage.save('tasks', toDoList.taskList);
+  });
+}
+
+function populateList(emptyList) {
+  emptyList.sortTasks();
+  for (let i = 0; i < emptyList.taskList.length; i += 1) {
+    const temp = createTaskDOM(emptyList.taskList[i].description);
+    taskList.appendChild(temp);
+    addCheckBoxListener(temp.children[0]);
+  }
+}
```

Comment:
This might nice to try with a forEach

Reply:
Thanks that is true. It will be easier to read. 👍